dist/button/ds6/button.css

```diff
@@ -310,24 +338,91 @@ a.fake-btn--secondary[aria-disabled="true"] {
   border-color: #c7c7c7;
   color: #c7c7c7;
 }
+button.btn--small,
+button.expand-btn--small,
+a.fake-btn--small,
+a.cta-btn--small {
+  padding-top: 5.5px;
+  padding-bottom: 5.5px;
+}
+button.btn--medium,
+button.expand-btn--medium,
+a.fake-btn--medium,
+a.cta-btn--medium {
+  padding-top: 9.5px;
+  padding-bottom: 9.5px;
+}
+button.btn--large,
+button.expand-btn--large,
+a.fake-btn--large,
+a.cta-btn--large {
+  padding-top: 11px;
+  padding-bottom: 11px;
+}
+button.btn--extra-large,
+button.expand-btn--extra-large,
+a.fake-btn--extra-large,
+a.cta-btn--extra-large {
+  padding: 13.5px 24px;
+}
 button.btn--large,
 button.expand-btn--large,
 a.fake-btn--large,
+a.cta-btn--large,
 button.btn--extra-large,
 button.expand-btn--extra-large,
-a.fake-btn--extra-large {
+a.fake-btn--extra-large,
+a.cta-btn--extra-large {
   font-size: 1.125rem;
 }
+button.btn--fixed-height,
+button.expand-btn--fixed-height,
+a.fake-btn--fixed-height,
+a.cta-btn--fixed-height {
+  height: 40px;
+}
+button.btn.btn--fixed-height.btn--small {
```

Comment:
BEM modifier classes should have no knowledge of each other (to avoid messy multi-inheritance woes). I wonder if we should used `btn--small-fixed`, `btn--medium-fixed`, etc. instead?

Reply:
Would `btn--small-fixed` include all `btn--small` rules? Or would they simply include new rules for being fixed?

Reply:
Because I could see this:

```
class="btn btn--regular btn--small btn--small-fixed"
```

Reply:
I was initially thinking it would include all small rules, but either way is fine.